### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by restropiajr**
Recording 2023-05-29 at 13 55 42

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,30 +1,60 @@
...
   <link rel="stylesheet" href="css/styles.css">
 </head>
+
 <body>
+
   <header>
-    <div class="container">
+    <div class="container column-full">
+      <h1>Code Journal</h1>
+      <h2>New Entry</h2>
```
Comment (shawnkost) [inline index.html]: This `h2` should be inside your `div` with `data-view="entry-form"`

File: index.html [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
+
 <body>
+
   <header>
-    <div class="container">
+    <div class="container column-full">
+      <h1>Code Journal</h1>
```
Comment (shawnkost) [inline index.html]: You should have some `div`s in-between here. It should go `container` > `row` > `column`. So I would do something like this.
```
<div class="container">
    <div class="row">
        <div class="column-full">
            <h1>Code Journal</h1>
        </div>
    </div>
</div>

File: index.html [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+      <h2>New Entry</h2>
     </div>
   </header>
-  <main class="container">
-    <div data-view="entry-form"></div>
-    <div data-view="entries" class="hidden"></div>
+
+  <main>
+    <div class="container column-full">
+      <div data-view="entry-form">
```
Comment (shawnkost) [inline index.html]: I would give the `container` to your `main`, and delete this `div` on line 30. You also shouldn't have a `column` that's not inside a `row`. So an element shouldn't both be a `container` & `column-full`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Roland! Also one more change that wasn't commented

1. The blue background in the `navbar` should never stop. It should reach the full width of the page.

File: index.html [hunk 4]
```diff
@@ -20,21 +20,25 @@
...
+      </div>
     </div>
   </header>
 
-  <main>
-    <div class="container column-full">
-      <div data-view="entry-form">
+  <main class="container">
+    <div class="row">
+      <div class="column-full" data-view="entry-form">
```
Comment (shawnkost) [inline index.html]: There should be no `div` in-between `main` & `div` with `data-view="entry-form"`. All `row` & `column` elements should be INSIDE your `div` with `data-view="entry-form"`. The div on line 34 should be the outermost div. Once we eventually get to view swapping, we want to hide everything inside this `div`, which is why we want all content related to this form to be inside of this `div`. So your `row` & `column-full` classes should be on `div`s inside of this `div` with `data-view="entry-form"`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here Roland!

**Comment by restropiajr** [comment]
Recording 2023-05-30 at 09 51 54

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**